specs/WebResourceRequested-CustomScheme.md

```diff
@@ -70,12 +70,14 @@ options.CustomSchemeRegistrations.Add(
   new CoreWebView2CustomSchemeRegistration(customScheme)
   {
     TreatAsSecure = true,
-    AllowedOrigins = { "https://*.example.com" }
+    AllowedOrigins = { "https://*.example.com" },
+    HasAuthorityComponent = true
   });
 options.CustomSchemeRegistrations.Add(
   new CoreWebView2CustomSchemeRegistration(customSchemeNotInAllowedOrigins)
   {
-    TreatAsSecure = true
+    TreatAsSecure = true,
+    HasAuthorityComponent = true
   });
```

Comment:
consider adding a new example CoreWebView2CustomSchemeRegistration where HasAuthorityComponent  is 'false' (left unset?) demonstrating default usage

Reply:
Good suggestion